fedmsg_meta_fedora_infrastructure/compose.py

```diff
@@ -27,88 +27,125 @@ class ComposeProcessor(BaseProcessor):
     __docs__ = "https://pagure.io/docs/releng"
     __obj__ = "Composes"
 
+    # dicts mapping topic substrings to subtitle event texts
+    # These are the old 'make-updates' topics
+    makeupevents = {
+        '.make-updates.':       "",
```

Comment:
For these dictionaries, do we have anything that'd be a little more specific, maybe ``releng.make-updates.`` & so on? I fear (down the road) a little the conflicts on topics if we stay that generic

Reply:
AIUI, the messages each processor operates on are determined by its `__name__` property. This processor will only ever be used for `compose.*` messages. Let me know if this is wrong.